akka-javasdk-tests/src/test/java/akkajavasdk/SdkIT.java

```diff
@@ -51,7 +51,7 @@
 import static org.assertj.core.api.Assertions.assertThat;
 
 @ExtendWith(Junit5LogCapturing.class)
-public class SdkIntegrationTest extends TestKitSupport {
```

Comment:
Let's revert them all back to `*IntegrationTest` as per other comment.

Reply:
Alright.